### PR TITLE
[ FEATURE ] Added Invite Functionality

### PR DESCRIPTION
**Pull request by prafulnigam**
Screenshot (476)

### CODE REVIEW

**Comment by prafulnigam** [comment]
Added the Functionality, Pleasure lookout for updates.

File: src/components/shared/LeftSidebar.tsx [hunk 1]
```diff
@@ -11,9 +11,39 @@ const LeftSidebar = () => {
...
-  }, [isSuccess]);
+  }, [isSuccess, navigate]);
+
+  const handleInviteClick = () => {
+    const currentURL = "https://opengram.vercel.app/sign-in";
+    navigator.clipboard.writeText(`Hey! I am inviting you to join this amazing Social Media Application: ${currentURL}`)
+      .then(() => {
+        const popup = document.createElement('div');
+        popup.textContent = "Invitation copied to clipboard!";
+        popup.style.cssText = `
```
Comment (Nishitbaria) [inline src/components/shared/LeftSidebar.tsx:25]: Hey we are using Tailwind css Can you please convert these changes?

Comment (Nishitbaria) [reply src/components/shared/LeftSidebar.tsx:25]: also, Can you fix the UI   : 

image

Its not look Good 

take reference of this  Figma file : [Link](https://www.figma.com/file/y7OzDDuGVslwr9rOYDQ7i2/OpenGrame?type=design&node-id=0%3A1&mode=design&t=H6AOHJuuNXj4Xrnk-1)

**Comment by prafulnigam** [comment]
Okay cool, On it !

**Comment by Nishitbaria** [comment]
> Okay cool, On it !

Any update @prafulnigam

**Comment by prafulnigam** [comment]
heyas you'll get the updated PR by noon

**Pull request closed without merging**